src/main/scala/com/gradle/internal/CustomBuildScanEnhancements.scala

```diff
@@ -1,328 +1,369 @@
 package com.gradle.internal
 
 import com.gradle.develocity.agent.sbt.api.configuration.BuildScan
-import com.gradle.internal.CiUtils.*
-import com.gradle.internal.Utils.*
-import sbt.URL
+import com.gradle.develocity.agent.sbt.api.configuration.Server
+import com.gradle.internal.Utils.Env
+import java.util.Properties
+import java.net.URL
+import sbt.Logger
 
 /**
  * Adds a standard set of useful tags, links and custom values to all build scans published.
  */
-class CustomBuildScanEnhancements(serverConfig: ServerConfigTemp, scalaVersions: String) {
-
-  private val buildScan = new BuildScanTemp()
-
-  private val SYSTEM_PROP_IDEA_VENDOR_NAME = "idea.vendor.name"
```

Comment:
can we keep the constants at the top of the file to keep more consistency?

Reply:
can we do this for all of the constants?

Reply:
we're missing this for the rest of the constants. let's merge this and then do it post merge.